pkg/cmd/server/origin/master.go

```diff
@@ -250,23 +247,11 @@ func (c *MasterConfig) RunHealth() error {
 	genericroutes.Profiling{}.Install(postGoRestfulMux)
 	genericroutes.MetricsWithReset{}.Install(postGoRestfulMux)
 
-	// TODO: replace me with a service account for controller manager
-	tokenReview := clientgoclientset.New(c.PrivilegedLoopbackKubernetesClientsetInternal.Authentication().RESTClient()).AuthenticationV1beta1().TokenReviews()
-	authn, err := serverauthenticator.NewRemoteAuthenticator(tokenReview, c.APIClientCAs, 5*time.Minute)
-	if err != nil {
-		return err
-	}
-	sarClient := clientgoclientset.New(c.PrivilegedLoopbackKubernetesClientsetInternal.Authorization().RESTClient()).AuthorizationV1beta1().SubjectAccessReviews()
-	remoteAuthz, err := authzwebhook.NewFromInterface(sarClient, 5*time.Minute, 5*time.Minute)
-	if err != nil {
-		return err
-	}
-
 	// we use direct bypass to allow readiness and health to work regardless of the master health
-	authz := serverhandlers.NewBypassAuthorizer(remoteAuthz, "/healthz", "/healthz/ready")
+	authz := serverhandlers.NewBypassAuthorizer(c.Authorizer, "/healthz", "/healthz/ready")
```

Comment:
I think this is only used by the controllers, but the construct built here where authorizer can mean different things is really undesireable.